org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -453,14 +453,14 @@ private void configureDetectors(Map<String, Object> dmodels, ScanModel model, IP
 				}
 				ret.add(detector);
 			}
-			
+
 			return ret;
-			
+
 		} catch (ScanningException ne) {
 			throw new EventException(ne);
 		}
 	}
-	
+
 	private List<IScannable<?>> getScannables(Collection<String> scannableNames) throws EventException {
 		// used to get the monitors and the metadata scannables
 		if (scannableNames==null) return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)